### DISCUSSION TITLE
multiple IP address

### DISCUSSION THREAD

**Discussion by FreeVB** [jchristn/SuperSimpleTcp#198] [Q&A]
Does it support listening on multiple IP address levels at the same time? For example, listen to all available IP addresses on the server at the same time.
For example: _Server = new SimpleTcpServer(new IPHost[] { new(7001), new(8089) })

**Comment by FreeVB** [comment]
The client can connect to any IP address on the server, such as 192.168.1.10, 8089, 216.16.XX.XX, 8089

One of the two IPs is a local area network IP and the other is an Internet IP。

**Comment by jchristn** [comment]
The library does not.  You'll need to start multiple instances of the server to do that.
